_javascript/lib/sidebarNavigation.js

```diff
@@ -10,22 +10,36 @@ export default (links) => {
     return element.getBoundingClientRect().top + window.pageYOffset - yOffset;
   };
 
-  const scrollToLink = (element) => {
-    const locationID = element.getAttribute("data-target");
+  const scrollToLink = (link) => {
+    const locationID = link.getAttribute("data-target");
     const targetElement = document.getElementById(locationID);
+    const y = getElementLocation(targetElement);
 
     window.scrollTo({
-      top: getElementLocation(targetElement),
+      top: y,
       behavior: "smooth",
     });
 
-    if (window.innerWidth < 768) {
-      const body = document.querySelector("body");
+    const intervalID = setInterval(function () {
+      // Check if the Y Offset has got within 10 pixels of the target
+      // it's not always going to be exact, so make sure we're roughly
+      // there
+      if (Math.abs(y - window.pageYOffset) < 5) {
+        link.blur();
+        targetElement.setAttribute("tabindex", "-1");
```

Comment:
What's setting the `tabindex` do?

Reply:
By default, only links and form elements are focusable. The only way to make headings focusable is by setting a negative `tabindex`

Reply:
Could you add an explanation to the commit message maybe?

Reply:
I've added a comment to the code, just to make it clearer